app/assets/stylesheets/application.css

```diff
@@ -51,6 +46,10 @@
   background-color: #ffdddd;
 }
 
+.compartial {
+  margin-left: 20px;
+}
```

Comment:
What is compartial standing for here? Comments partial? If that's the case, it might be good to give it a more explicit name. Also, someone with more FE experience might have some thoughts on how to do margin like this.